<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createCurrentAccount</class>
 <widget class="QWidget" name="createCurrentAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>createCurrentAccount</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>480</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
   <property name="text">
    <string>Mobile number</string>
   </property>
  </widget>
  <widget class="QLabel" name="instructionlabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
   <property name="text">
    <string>Please fill the following form :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>450</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_7">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>530</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="resultlabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>380</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
   <property name="text">
    <string>Address</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
   <property name="text">
    <string>Age</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createAccountButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>600</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
   <property name="text">
    <string>create current account</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>450</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
   <property name="text">
    <string>Phone number</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>530</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
   <property name="text">
    <string>Balance deposited</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>480</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>380</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>180</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
   <property name="text">
    <string>CNIC</string>
   </property>
  </widget>
  <widget class="QLabel" name="accountlabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
   <property name="text">
    <string>Name (as per CNIC)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>330</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
   <property name="text">
    <string>Occupation</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="accountNoEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="heading">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Create Current Account</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
   <property name="text">
    <string>Date of Birth</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>330</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>280</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
    </font>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>-55</x>
     <y>-109</y>
     <width>911</width>
     <height>821</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/guidesign/BackgroundImage.png);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>600</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <zorder>graphicsView</zorder>
  <zorder>label_7</zorder>
  <zorder>instructionlabel</zorder>
  <zorder>lineEdit_3</zorder>
  <zorder>lineEdit_7</zorder>
  <zorder>resultlabel</zorder>
  <zorder>label_5</zorder>
  <zorder>label_2</zorder>
  <zorder>createAccountButton</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>label_6</zorder>
  <zorder>label_8</zorder>
  <zorder>lineEdit_6</zorder>
  <zorder>lineEdit_5</zorder>
  <zorder>lineEdit</zorder>
  <zorder>label</zorder>
  <zorder>accountlabel</zorder>
  <zorder>label_4</zorder>
  <zorder>accountNoEdit</zorder>
  <zorder>heading</zorder>
  <zorder>label_3</zorder>
  <zorder>lineEdit_4</zorder>
  <zorder>dateEdit</zorder>
  <zorder>pushButton</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="bankingsystem.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>createAccountButton</sender>
   <signal>clicked()</signal>
   <receiver>createCurrentAccount</receiver>
   <slot>onCreateCurrentAccountClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>611</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>createCurrentAccount</receiver>
   <slot>goBack()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>517</x>
     <y>611</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onCreateCurrentAccountClicked()</slot>
  <slot>goBack()</slot>
 </slots>
</ui>
